Source/SlashCmd/Main.lua

```diff
@@ -0,0 +1,54 @@
+local _, addonTable = ...;
+local zc = addonTable.zc;
+
+function Auctionator.SlashCmd.Initialize()
+  SlashCmdList["Auctionator"] = Auctionator.SlashCmd.Handler
+  SLASH_Auctionator1 = "/auctionator"
+  SLASH_Auctionator2 = "/atr"
+end
+
+function Auctionator.SlashCmd.Handler(input)
```

Comment:
I had some typos that I think made my last comment unclear. Here's my proposal (a single dispatch table with all commands, and explicit handlers for each command, rather than parameters to commands):

```lua
local _, addonTable = ...;
local zc = addonTable.zc;

function Auctionator.SlashCmd.Initialize()
  SlashCmdList["Auctionator"] = Auctionator.SlashCmd.Handler
  SLASH_Auctionator1 = "/auctionator"
  SLASH_Auctionator2 = "/atr"
end

local SLASH_COMMANDS = {
  ["r"] = Auctionator.Variables.ResetTimer,
  ["reset"] = Auctionator.Variables.ResetTimer,
  ["db"] = Auctionator.Variables.ResetDatabase,
  ["database"] = Auctionator.Variables.ResetDatabase,
  ["d"] = Auctionator.Variables.ToggleDebug,
  ["debug"] = Auctionator.Variables.ToggleDebug
}

function Auctionator.SlashCmd.Handler(input)
  Auctionator.Debug.Message( 'Auctionator.SlashCmd.Handler', input )

  local command = {zc.words(input:lower())};

  if #command == 0 then
    return
  end

  local handler = SLASH_COMMANDS[command[1]]
  if handler == nil then
    Auctionator.Utilities.Message("Unrecognized command '" .. command[1] .. "'")
    return
  end
  
  handler(command[2])
end

function Auctionator.Variables.ResetTimer()
  if Auctionator.Debug.IsOn() then
    Auctionator.FullScan.State.TimeOfLastScan = nil
    Auctionator.Utilities.Message("Scan timer reset.")
  else
    Auctionator.Utilities.Message("Requires debug mode.")
  end
end

function Auctionator.Variables.ResetDatabase()
  if Auctionator.Debug.IsOn() then
    AUCTIONATOR_PRICE_DATABASE = nil
    Auctionator.Variables.InitializeDatabase()
    Auctionator.Utilities.Message("Price database reset.")
  else
    Auctionator.Utilities.Message("Requires debug mode.")
  end
end

function Auctionator.Variables.ToggleDebug()
    Auctionator.Debug.Toggle()

    if Auctionator.Debug.IsOn() then
      Auctionator.Utilities.Message("Debug mode on.")
    else
      Auctionator.Utilities.Message("Debug mode off.")
    end
end
```